java/server/src/org/openqa/selenium/grid/graphql/Node.java

```diff
@@ -59,6 +58,22 @@ public int getMaxSession() {
     return maxSession;
   }
 
+  public StringBuilder getCapabilities() {
+    StringBuilder str = new StringBuilder("[");
```

Comment:
We could simplify this greatly by having this method return a Map of what we want it to hold and the call `JSON.toJson` on our new map

Reply:
I'm a bit confused, what would be the key and value of map in that case?